### ISSUE TITLE
Delegate Methods

### ISSUE THREAD

**Issue by foffux** [apptality/ATAppUpdater#11]
Hello
What about developing also delegate methods?
Like those ones:

``` javascript
- (void)atAppUpdaterDidShowUpdateDialog
- (void)atAppUpdaterUserDidLaunchAppStore
- (void)atAppUpdaterUserDidSkipVersion
- (void)atAppUpdaterUserDidCancel
```

**Comment by brianjd** [comment]
Sounds like a great idea to me.

**Comment by emotality** [comment]
I agree :smile: Will do it asap, thanks!

**Comment by foffux** [comment]
Hi, I do not understand. Are you working on that? How long does it take you, at about?
Thank you.

**Comment by emotality** [comment]
I'm currently busy with work stuff, will do it when I can.

**Comment by jameshays** [comment]
I'd much prefer this moved to using UIAlertController and expose UIAlertAction.  I'll create a PR for this since I'm needing it for a local project.

**Comment by emotality** [comment]
Sorry for long wait guys, been super busy with full-time work and other personal projects. The above is fixed with commit dc16fe76ba81be6372323c7b24625559c6d3f9b3

Thanks for reporting issues!
